test/parallel/test-fs-realpath.js

```diff
@@ -185,7 +186,7 @@ function test_deep_relative_dir_symlink(callback) {
     common.printSkipMessage('symlink test (no privs)');
     return runNextTest();
   }
-  const expected = path.join(common.fixturesDir, 'cycles', 'folder');
+  const expected = path.join(fixtures.fixturesDir, 'cycles', 'folder');
```

Comment:
There is the `fixtures.path` function for this and it would be nice to use that instead.